### PR TITLE
fix(nms): Fix some console errors

### PR DESCRIPTION
**Pull request by thmsschmitt**
## Summary
Fix some errors in the development console.

## Test Plan
Open the relevant pages and make sure they still look ok.

### CODE REVIEW

File: nms/server/grafana/routes.tsx [hunk 1]
```diff
@@ -116,7 +116,7 @@ const proxyMiddleware = () => {
       },
       proxyReqPathResolver: req => req.originalUrl.replace(/^\/grafana/, ''),
       userResDecorator: (proxyRes, proxyResData: Buffer, userReq, userRes) => {
-        userRes.set('X-Frame-Options', 'allow');
```
Comment (thmsschmitt) [inline nms/server/grafana/routes.tsx:119]: Allow is not a valid option https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options

File: nms/app/components/ActionTable.tsx [hunk 2]
```diff
@@ -55,6 +55,7 @@ const tableIcons = {
   Add: forwardRef<SVGSVGElement>((props, ref) => (
     <Button
       startIcon={<AddCircleOutlined {...props} ref={ref} />}
+      component="span"
```
Comment (thmsschmitt) [inline nms/app/components/ActionTable.tsx:58]: prevents illegal nesting

File: nms/app/views/equipment/GatewayPoolGatewaysEdit.tsx [hunk 3]
```diff
@@ -160,7 +160,10 @@ export default function GatewayEdit(props: GatewayPoolEditProps) {
           )}
           {gateways.length > 0 &&
             gateways.map((gw, index) => (
-              <ListItem component={Paper} className={classes.gatewayPrimary}>
+              <ListItem
+                component={Paper}
+                className={classes.gatewayPrimary}
+                key={index}>
```
Comment (thmsschmitt) [inline nms/app/views/equipment/GatewayPoolGatewaysEdit.tsx:166]: missing key

File: nms/app/views/subscriber/SubscriberAddDialog.tsx [hunk 4]
```diff
@@ -54,10 +54,6 @@ import type {
 } from './SubscriberUtils';
 
 const useStyles = makeStyles(() => ({
-  dialogTitle: {
```
Comment (thmsschmitt) [inline nms/app/views/subscriber/SubscriberAddDialog.tsx:57]: This was never applied correctly

File: nms/app/views/domain-proxy/LogsList.tsx [hunk 5]
```diff
@@ -142,12 +142,12 @@ function LogsList() {
   const [fccId, setFccId] = useState<string>('');
   const [from, setFrom] = useState<LogsDirectionNullable>(null);
   const [to, setTo] = useState<LogsDirectionNullable>(null);
-  const [responseCode, setResponseCode] = useState<string | null>(null);
+  const [responseCode, setResponseCode] = useState<string>('');
```
Comment (thmsschmitt) [inline nms/app/views/domain-proxy/LogsList.tsx:145]: `null` was passed a value for an input

**Review by Neudrino — APPROVED** [review]

**Pull request merged**